sass/patternfly-react.scss

```diff
@@ -2,22 +2,23 @@
   Third Party / Dependency imports here
 */
 $font-path:      '~patternfly/dist/fonts/';
-$img-path:        '~patternfly/dist/img/';
+$img-path:       '~patternfly/dist/img/';
 $icon-font-path: '~patternfly/dist/fonts/';
 
-
 // Bootstrap Core variables and mixins
-@import "bootstrap/variables";
-@import "bootstrap/mixins";
+@import "@bootstrap/variables";
```

Comment:
Creating a special syntax will make it difficult to import this file for consumers.
I have an open pr for foreman that handle it differently:
https://github.com/theforeman/foreman/pull/5212/files#diff-a58d55bdb5770c78ad512f8e91f8d051R109

Reply:
I don't expect consumers would import this file, rather they would import the [patternfly-react partials](https://github.com/patternfly/patternfly-react/blob/master/sass/patternfly-react/_patternfly-react.scss). But if there is a better way to avoid the special syntax, that would be better.

Reply:
oh you right, it's a bit confusing since they use they same name

Reply:
Really the reason I put the "@" there was to indicate that the import is coming from somewhere non-standard.  In this case, the bootstrap and fontawesome imports all come from the patternfly package.  That was achieved by including those source sass directories directly on the command line.

I though about changing the import to something that webpack would understand natively like:

`@import "~patternfly/node_modules/bootstrap-sass/assets/stylesheets/variables" ;`

but that seemed worse.